LexiconSite/searchTesting.py

```diff
@@ -50,13 +54,37 @@ def search_result_narrow(search_term: str, table: Table) -> str:
 
     return results
 
+def download_images(search_result: list):
```

Comment:
may be worth leaving a comment / docstring here that describes the expected structure of the list?